src/Dfc.CourseDirectory.Core/DataStore/CosmosDb/Models/Course.cs

```diff
@@ -10,6 +10,11 @@ public class Course
     {
         [JsonProperty("id")]
         public Guid Id { get; set; }
+        /// <summary>
+        /// CosmosDb ETag for optimistic concurrency
+        /// https://github.com/Azure/azure-cosmos-dotnet-v2/blob/master/samples/code-samples/DocumentManagement/Program.cs
+        /// </summary>
+        public string _etag { get; set; }
```

Comment:
Unnecessary comment?

Reply:
I think it deserves some kind of pointer to it being a cosmos thing.

Reply:
Fair enough. Can you not used a convention correct property name and `JsonProperty`?

Reply:
Ah, fab, didn't think of that. Thanks! I'll do that

Reply:
done